developer_guides/debugability/coredump-reader/index.rst

```diff
@@ -0,0 +1,35 @@
+.. _dbg-coredump-reader:
+
+Coredump-reader
+###############
+
+Tool for processing FW stack dumps. In verbose mode it prints DSP registers,
+and function call addresses in stack up to that which caused core dump. It
+then prints either to file or to stdin all gdb commands unwrapping those
+function call addresses to function calls in human readable format.
+
+Coredump-reader usage
+*********************
+
+Usage sof-coredump-reader.py [-h] [-a ARCH] [-c] [-l COLUMNCOUNT] [-v] (--stdout | -o OUTFILE) [--stdin | -i INFILE]
+
+-h				show this help message and exit
+-a ARCH			determine architecture of dump file; valid archs are: LE64bit, LE32bit
+-c				set output to be colourful
+-l COLUMNCOUNT	set how many colums to group the output in
+-v				increase output verbosity
+--stdin			input is from stdin
+-i INFILE		path to sys dump bin
+--stdout		output is to stdout
+-o OUTFILE		output is to FILE
+
+
+
+- sof-coredump-to-gdb.sh
+
+Exemplatory usage of sof-coredump-reader.py
```

Comment:
Line 30: "Example usage of ..."

Reply:
Also, this says usage for sof-coredump-reader.py, but he command is for sof-coredump-to-gdb.sh. It's a little confusing.

Reply:
Thats because sof-coredump-to-gdb.sh is a script which shows (inside) example usage of .py file and we can just call this script to test it.
I have changed this a little, maybe it will fit better now